beacon-chain/types/block.go

```diff
@@ -12,6 +12,39 @@ import (
 	"golang.org/x/crypto/blake2b"
 )
 
+// BlockRegistry is a struct used to stor the blockhashes of blocks saved in the DB.
+type BlockRegistry struct {
+	data *pb.BlockRegistry
```

Comment:
Why is a block registry a proto? It is not something we would ever sync between nodes or send via p2p. The only purpose is to keep a representation of the latest block hashes that are pending fork-choice rule. Let's just use a regular Go struct instead.